RedHatInsights/compliance-frontend
# feat(CreatePolicy): RHICOMPL-2590 - Use Table for policy type selection

This updates the policy type selection in the "Create Policy" wizard to also include the "Supported minor versions":

**Before:**

![Screenshot 2022-01-28 at 02 11 43](https://user-images.githubusercontent.com/7757/151469526-15b85a2e-4793-4e13-9e9c-3e7c9eeb3a3f.png)

**After:**

![Screenshot 2022-01-28 at 00 12 11](https://user-images.githubusercontent.com/7757/151469554-513b3e78-a8b3-40ca-b69b-347fda206fb2.png)


**How to test:**

1. Open the "Create Policy" wizard on the "SCAP Policies" page
2. Verify that it is in fact a table now and it works as other tables
3. Verify that selecting any policy works
4. Verify that it sets the values properly, by completing the wizard



## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
